File: Wrap Up/JavaScript/PerfectNumber.js
const prompt = require('prompt-sync')();

let number = Number(prompt("Enter a number: "));

let sum = 0;
for (let count = 1; count < number; count++) {
    if (number % count === 0) {
        sum = sum + count;
    }
}

if (sum === number) {
    console.log(number + " is a Perfect Number.");
} else {
    console.log(number + " is NOT a Perfect Number.");
}

File: Wrap Up/Java/PerfectNUmber.java
import java.util.Scanner;

public class PerfectNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int number = scanner.nextInt();

        int sum = 0;
        for (int count = 1; count < number; count++) {
            if (number % count == 0) {
                sum = sum + count;
            }
        }

        if (sum == number) {
            System.out.println(number + " is a Perfect Number.");
        } else {
            System.out.println(number + " is NOT a Perfect Number.");
        }
    }
}
